pkg/config/config.go

```diff
@@ -869,6 +872,15 @@ func ResolveSecrets(config Config, origin string) error {
 	return nil
 }
 
+// Read the API from a file
+func readAPIKeyFromFile(config Config) {
+	if apiKeyFile := config.GetString("api_key_file"); apiKeyFile != "" {
+		if apiKey, err := ioutil.ReadFile(apiKeyFile); err == nil {
+			config.Set("api_key", apiKey)
```

Comment:
we should log the error if there's one, to help troubleshooting perm issues for example.

Even if the logger isn't set up yet when this is called, there's a mechanism to buffer entries and output them once the logger is set up.

Reply:
Log added